### ISSUE TITLE
"Base tree for Groups is empty, using Base DN"

### ISSUE THREAD

**Issue by LukasReschke** [nextcloud/server#2585]
I get like hundreds of these entries in my debug log in a single second. Are they any useful, @blizzz?

**Comment by blizzz** [comment]
@LukasReschke: no, I am also annoyed and was so close to kicking it out last week. Something distracted me.
